packages/backend-common/src/service/lib/ServiceBuilderImpl.ts

```diff
@@ -228,6 +228,10 @@ export function applyCspDirectives(
   const result: ContentSecurityPolicyOptions['directives'] =
     helmet.contentSecurityPolicy.getDefaultDirectives();
 
+  // TODO(Rugvip): We currently use non-precompiled AJV for validation in the frontend, which uses eval.
+  //               It should be replaced by any other solution that doesn't require unsafe-eval.
+  result['script-src'] = ["'self'", "'unsafe-eval'"];
```

Comment:
Not that it matters too much to us, but I just had to go double check that they actually make a copy :) And not a deep copy apparently - so using push here would actually have been a risk. Ah well, not worth making a deep copy defensively

Reply:
Yep double checked that as well for the previous change, so knew to avoid `.push` x)